tests/Struct/FindologicProductTest.php

```diff
@@ -552,7 +552,11 @@ private function getAttributes(ProductEntity $productEntity): array
                     ->getName()
             ]
         );
-        $attributes[] = new Attribute('shipping_free', [$productEntity->getShippingFree() ? 1 : 0]);
+
+        $translationKey = $productEntity->getShippingFree() ? "finSearch.general.yes" : "finSearch.general.no";
+        $shippingFree = $this->getContainer()->get('translator')->trans($translationKey);
+
```

Comment:
That newline should be put after the assignment of `shipping_free` to `$attributes`. Here it does not separate anything.